collaboration/learning_goals.md

```diff
@@ -2,4 +2,60 @@
 
 ## Collective
 
+### Improve Collaboration Skills  
+
+Our goal is to work together as a team by using Git to manage our code.  
+We will learn how to collaborate effectively and resolve conflicts.
+
+### Master Version Control  
+
+We want everyone on the team to become comfortable with using Git.  
+This includes learning how to make changes, create branches, and manage pull requests.
+
+### Understand Team Dynamics  
+
+Our goal is to support all team members, no matter their skill level.  
+We will encourage everyone to participate equally and contribute to the project.
+
+### Review Code Collaboratively  
+
+Our goal is to learn how to review and provide constructive feedback on code.
+
+### Learning From Each Other  
+
+Our goal is to learn various problem-solving strategies from each other’s code.
+We will focus on understanding how and why someone solved a problem in that manner.
+
+### Improving coding skills
+
+Our goal is to enhance our Python skills.
+We will learn Python commands
+and explore how to apply them effectively to solve problems.
+
+### Improve Unit tests
+
+Our goal is to enhance our skills in unit testing.
+We will learn from the unit tests written by our group members,
+understanding how they apply to our specific cases and improving our approach.
+
 ## Individual
```

Comment:
**ADD YOUR NAME AS YOU ARE DISCUSSING INDIVIDUAL GOALS**

Reply:
Name has been added under individual